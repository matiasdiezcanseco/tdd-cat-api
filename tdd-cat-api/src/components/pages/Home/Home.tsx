import Layout from '../../templates/Layout/Layout'

const Home = () => {
  return (
    <Layout>
      <div
        data-testid="homepage-testid"
        className="flex flex-col items-center justify-center h-full"
      >
        <h1 className="text-5xl">Un lugar para los amantes de los gatos</h1>
        <p className="text-2xl">Encuentra tus razas favoritas</p>
      </div>
    </Layout>
  )
}

export default Home
