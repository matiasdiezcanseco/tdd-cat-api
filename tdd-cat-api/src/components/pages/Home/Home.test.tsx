import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'
import Home from './Home'

describe('test Home', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    )
    expect(container).toMatchSnapshot()
  })
})
