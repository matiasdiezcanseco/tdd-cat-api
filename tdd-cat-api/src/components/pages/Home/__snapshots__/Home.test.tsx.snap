// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Home should match snapshot 1`] = `
<div>
  <div
    class="flex flex-col h-screen"
  >
    <div
      class="py-4 bg-slate-500"
    >
      <ul
        class="flex justify-center space-x-40 text-gray-100 text-lg"
      >
        <li
          class="cursor-pointer hover:underline"
        >
          Home
        </li>
        <li
          class="cursor-pointer hover:underline"
        >
          Buscar
        </li>
      </ul>
    </div>
    <div
      class="flex-grow bg-gray-300"
    >
      <div
        class="flex flex-col items-center justify-center h-full"
        data-testid="homepage-testid"
      >
        <h1
          class="text-5xl"
        >
          Un lugar para los amantes de los gatos
        </h1>
        <p
          class="text-2xl"
        >
          Encuentra tus razas favoritas
        </p>
      </div>
    </div>
  </div>
</div>
`;
